tests/dml_vdm/dml_vdm.c

```diff
@@ -0,0 +1,45 @@
+// SPDX-License-Identifier: BSD-3-Clause
+/* Copyright 2022, Intel Corporation */
+
+#include <assert.h>
+#include <stdlib.h>
+#include <stdio.h>
+#include <string.h>
+
+#include <libminiasync.h>
+#include <libminiasync-dml.h>
+
+int
+dml_basic_memcpy()
+{
+	char *buf_a = strdup("testbuf");
+	char *buf_b = strdup("otherbuf");
+	size_t copy_size = strlen(buf_a);
+
+	struct runtime *r = runtime_new();
+
+	struct vdm *dml_mover_async = vdm_new(vdm_descriptor_dml());
```

Comment:
how can we test both hw/sw paths?
(In this case we do not know if your switch/translation works (?))